x-pack/legacy/plugins/siem/public/components/edit_data_provider/index.tsx

```diff
@@ -101,7 +101,7 @@ export const StatefulEditDataProvider = React.memo<Props>(
     const [updatedValue, setUpdatedValue] = useState<string | number>(value);
 
     /** Focuses the Value input if it is visible, falling back to the Save button if it's not */
-    function focusInput() {
+    const focusInput = () => {
```

Comment:
Any benefit to memoizing this as well? Since `onFieldSelected` & `onOperatorSelected` are memoized via `useCallback`, they'll only ever call the instance created by the initial render, correct? So all future `focusInput`'s initialized in subsequent renders would not be used?

Reply:
I'ma answer that question with a `no` after referencing the following: https://kentcdodds.com/blog/usememo-and-usecallback